docs/quickstart-minikube.md

```diff
@@ -176,9 +176,11 @@ Next we will create a local cluster using Minikube. You can also [try OSBA on th
 
 1. Create a Minikube Cluster:
     ```console
-    minikube start --bootstrapper=kubeadm
+    minikube start
     ```
 
+    Note: Service Catalog may not work with Kubernetes versions less than 1.9.x. If you are using a version of Minikube older than 0.26, you will need to upgrade to 0.26 to ensure the cluster is compatible with Service Catalog and OSBA.
```

Comment:
s/1.9.x/1.9.0/

s/0.26/0.25/... unless you want to omit that bit entirely, since I believe 0.25 would already have been listed elsewhere as the minimum version of mk. Dealer's choice on whether to mention here as well or not.